src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -44,6 +45,8 @@ private void handleCommandEntered() {
         try {
             commandExecutor.execute(commandText);
             commandTextField.setText("");
+        } catch (InvokeCommandException e) {
+            commandTextField.setText(e.getMessage());
         } catch (CommandException | ParseException e) {
             setStyleToIndicateCommandFailure();
```

Comment:
Is catching `InvokeCommandException` necessary? Since `InvokeCommandException` is a child of `CommandException` and will be caught in caught by the line `catch (CommandException | ParseException e)`.

Reply:
Yes, I need to separate InvokeCommandException in order to auto fill the content.